libcontainer/cgroups/systemd/common.go

```diff
@@ -349,3 +350,26 @@ func stopUnit(dbusConnection *systemdDbus.Conn, unitName string) error {
 	}
 	return nil
 }
+
+func addCpuQuota(properties *[]systemdDbus.Property, quota int64, period uint64) error {
+	// cpu.cfs_quota_us and cpu.cfs_period_us are controlled by systemd.
```

Comment:
and cpu.max

Reply:
right, this comment comes from cgroupv1, will fix